deploy/crds/base/jvmbuildservice.io_jbsconfigs.yaml

```diff
@@ -87,6 +87,13 @@ spec:
                 type: object
               enableRebuilds:
                 type: boolean
+              gitDeployment:
```

Comment:
Sorry I didn't pick this up before, but this is not really 'deployment'. Maybe gitSourceArchive would be a better name? I know this is minor but the CRDs need to be backwards compatible so changing things later is a pain.